
<launch>

  <group ns="datalink_channel_1">
    <node pkg="uav_control" name="datalink_ch1" type="datalink_ch1" />
  </group>



</launch>
